_releases/2022-01-06-1.3.0-released.md

```diff
@@ -22,6 +22,46 @@ In this release we included [XXX PRs since the 1.2.2 release](https://github.com
 
 ## Multi-Assign
 
+Multiple assignments got improved in various ways. First, it's possible to take a "splat of elements" in a multi-assignment ([#10410](https://github.com/crystal-lang/crystal/pull/10410)):
+
+<div class="code_section">{% highlight crystal %}
+# Splat in multi-assign with array
+first, *rest, last = [1, 2, 3, 4, 5]
+first # => 1
+rest # => [2, 3, 4]
+last # => 5
+{% endhighlight %}</div>
+
+<div class="code_section">{% highlight crystal %}
+# Splat in multi-assign with tuple
+*rest, last = {"This", 15, 4, "tuple", true}
+rest # => {"This", 15, 4, "tuple"}
+last # => true
+{% endhighlight %}</div>
+
+To take the first and last element, it is possible to use the _underscore splat_ notation:
+<div class="code_section">{% highlight crystal %}
+# Ignoring the elements in the middle
+first, *_, last = {"This", 15, 4, "tuple", true}
+first # => "This"
+last # => true
+{% endhighlight %}</div>
+
+Second, there's an optional preview feature to detect unbalanced multi-assignments ([#11145](https://github.com/crystal-lang/crystal/pull/11145)). It can be enabled with the compiler flag `-Dstrict_multi_assign`.
+<div class="code_section">{% highlight crystal %}
+a, b, c = {1, 2} # Error: index out of bounds for Tuple(Int32, Int32) (2 not in -2..1)
+a, b = {1, 2, 3} # Error: cannot assign Tuple(Int32, Int32, Int32) to 2 targets
+{% endhighlight %}</div>
+
+Note that the first example fails with a similar error in 1.2.2 (this error is [expected to change](https://github.com/crystal-lang/crystal/issues/11695) in the near future).
+
+To ignore the extra items in strict mode you can use the underscore splat notation:
```

Comment:
```suggestion
To ignore the extra items in strict mode you can use the underscore splat notation. This can be used to port existing code to work with the `strict_multi_assign` flag.
```